layouts/partials/collapsible-menu.html

```diff
@@ -21,38 +21,62 @@
 {{ end }}
 {{$currentSection := .Scratch.Get "currentSection" }}
 
+<!--- Sidebar layout --->
+{{ .Scratch.Set "sidebarLayout" "default" }}
+{{ if isset .Site.Params "sidebar_layout" }}
+  {{ .Scratch.Set "sidebarLayout" .Site.Params.sidebar_layout }}
+{{ end }}
+{{ if isset .Page.Params "sidebar_layout" }}
+  {{ .Scratch.Set "sidebarLayout" .Page.Params.sidebar_layout }}
+{{ end }}
+
+{{ $sidebarLayout := .Scratch.Get "sidebarLayout" }}
+
+{{ $classname := "collapsible-menu" }}
+{{ if eq $sidebarLayout "sidebar_block" }}
+  {{ $classname = "sidebar-block" }}
+{{ end }}
+
 <!----------------------- Collapisible start ------------------------------------>
   {{ range .Site.Menus.sidebar }}
     {{$parentItem := . }}
     {{ if and (.HasChildren) (eq $currentSection $parentItem.Identifier) }}
-    <div class="collapsible-menu container hidden-lg hidden-md visible-sm visible-xs">
-      <div class="container collapsible-menu-item related-content">
+    <div class="{{ $classname }} margin-top-10 container hidden-lg hidden-md visible-sm visible-xs">
+      <div class="container padding-left-0 padding-right-0 {{ $classname }}-item related-content">
         <a href="#sidebars-contents" data-toggle="collapse" aria-expanded="false" aria-controls="sidebars-contents">
           <div class="col-xs-6"><i class="fa fa-plus fa-bars"></i></div>
-          <div class="col-xs-18 collapsible-menu-text">Related Content</div>
+          <div class="col-xs-18 {{ $classname }}-text">Related Content</div>
         </a>
-      </div>  
-      <div class="container collapse" id="sidebars-contents">
+      </div>
+      <div class="container collapse {{ if eq $classname "sidebar-block" }} padding-left-0 padding-right-0 {{ end }}" id="sidebars-contents">
       {{ range .Children }}
         {{ if eq .HasChildren false }}
-          <div class="row collapsible-menu-item{{ if eq $currentPage.Title .Name }} active{{ end }}">
+          <div class="row {{ $classname }}-item{{ if eq (lower $currentPage.Title) (lower .Name) }} active{{ end }}">
```

Comment:
why are we adding to lower case comparisons here? Its expected that they should be exact matches. If we were to keep this code, you should store the output of the check at the start of the given loop so that we don't have to reformat the same text

Reply:
Copied from Eric. but I think you're right, we don't quite need it. 

But on second thought, probably just in case if someone adding pages and sidebar names forgot to keep the consistency of capitalizations?

Reply:
But this is checking each child within the loop. 🤔

Reply:
I'd honestly say we should scrap the lower check, as it could get expensive for larger sites. In the past we required exact match, and there was no request to change this functionality

Reply:
Sounds good, updated.